backend/app/services/chat/preprocessing/selected_documents.py

```
@@ -62,6 +62,61 @@
+    from app.services.group_permission import is_restricted_analyst
+
+    if not knowledge_base_ids:
+        return True, ""
+
+    # Get knowledge bases to check their namespaces
+    kbs = (
+        db.query(Kind)
+        .filter(
+            Kind.id.in_(list(knowledge_base_ids)),
+            Kind.kind == "KnowledgeBase",
+            Kind.is_active == True,
```

Ruff (0.15.5)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)